job.c

```diff
@@ -326,7 +326,7 @@ static void handle_cue_files(struct ptr_array *files)
 		if (!ents[i] || !is_cue(ents[i]))
 			continue;
 
-		char **files_in_cue;
+		char **files_in_cue = NULL;
 		int n = cue_get_files(ents[i], &files_in_cue);
```

Comment:
The uninitialized pointer happens when cue_get_files returns -1, which happens when cue_from_file returns NULL, which happens if mmap_file fails or the cue parse returns an error.

The fix should be to add an `if (n == -1) continue` after the cue_get_files call.